### DISCUSSION TITLE
Where could I know default autofix rules?

### DISCUSSION THREAD

**Discussion by ishowman** [eslint/eslint#15152] [Q&A] [answered]
I can't find it at docs.I don't know which rules can be auto fixed……

**Comment by aladdin-add** [comment, marked as answer]
https://eslint.org/docs/rules/

![image](https://user-images.githubusercontent.com/13050025/136929151-d6ad5461-1abe-45c8-8a1e-92c87434c05e.png)

**Reply by ishowman** [reply]
thanks for your reply.
